openmoc/plotter.py

```diff
@@ -515,6 +515,139 @@ def plot_flat_source_regions(geometry, gridsize=250, xlim=None, ylim=None):
 
 
 ##
+# @brief This method takes in a Geometry object and plots a color-coded 2D
+#        surface plot representing the flat source regions in the Geometry.
+#        The FSR centroids are plotted as black circles on top of the FSRs.
+# @details The Geometry object must be initialized with Materials, Cells,
+#          Universes and Lattices before being passed into this method. A user
+#          may invoke this function from an OpenMOC Python file as follows:
+#
+# @code
+#         openmoc.plotter.plot_centroids(geometry)
+# @endcode
+#
+# @param geometry a geometry object which has been initialized with Materials,
+#        Cells, Universes and Lattices
+# @param gridsize an optional number of grid cells for the plot
+# @param xlim optional list/tuple of the minimim/maximum x-coordinates
+# @param ylim optional list/tuple of the minimim/maximum y-coordinates
+def plot_centroids(geometry, gridsize=250, xlim=None, ylim=None):
+
+  global subdirectory
```

Comment:
After looking at this routine, is it really necessary? Perahsp an optional ``centroids=False`` keyword param for the ``plot_flat_source_regions(...)`` routine would better for code reuse?